client/a8c-for-agencies/sections/marketplace/controller.tsx

```diff
@@ -34,23 +33,13 @@ export const marketplaceProductsContext: Callback = ( context, next ) => {
 	context.primary = (
 		<>
 			<PageViewTracker title="Marketplace > Products" path={ context.path } />
-			{ isEnabled( 'a4a-product-page-redesign' ) ? (
-				<ProductsOverviewV2
-					siteId={ site_id }
-					suggestedProduct={ product_slug }
-					defaultMarketplaceType={ purchaseType }
-					productBrand={ getValidBrand( productBrand ) }
-					searchQuery={ search_query }
-				/>
-			) : (
-				<ProductsOverview
-					siteId={ site_id }
-					suggestedProduct={ product_slug }
-					defaultMarketplaceType={ purchaseType }
-					productBrand={ getValidBrand( productBrand ) }
-					searchQuery={ search_query }
-				/>
-			) }
+			<ProductsOverviewV2
```

Comment:
I will change the name of this component to ProductsOverview in a different pull request. Making would complicate the code review process and make it harder to follow.